docs/STUMP_Tiles_Algorithm_for_Cache_Optimization.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>I think diagonal computation wasn't in the Matrix Profile I paper.</p><p> </p><p>Maybe explain that the red box is the cache?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<blockquote>I think diagonal computation wasn't in the Matrix Profile I paper.</blockquote><p> </p><p>Yepp that's my bad, I believe the diagonal computation is in the Matrix Profile II paper?</p><p> </p><blockquote>Maybe explain that the red box is the cache?</blockquote><p> </p><p>So I've mentioned that the orange rectangle represents the cache. Should I provide a more detailed explanation?</p>

Reply:
<p>I think it was first introduced here: https://www.cs.ucr.edu/~eamonn/ten_quadrillion.pdf</p>

Reply:
<p>Ahh, I missed that you called out the orange box. I think you should mention it upfront and draw the readers attention to it. Right now, that information is presented AFTER </p>